src/components/common/maps/Map.js

```diff
@@ -72,34 +69,32 @@ class SimpleMap extends PureComponent {
   /*
     EVENTS
   */
-  _onClick = ({ x, y, lat, lng, event }) => console.log(x, y, lat, lng, event)
 
-  _onBoundsChange = (center, zoom /* , bounds, marginBounds */) => {
+  /* Parameters { x, y, lat, lng, event } */
+  _onClick = () => {}
+
+  /* Parameters center, zoom, bounds, marginBounds */
+  _onBoundsChange = (center, zoom) => {
     this.setState({ currentCenter: center });
```

Comment:
You can use `setState` once and have the two changes together.

Reply:
I really agree